mungegithub/mungers/approvers/owners.go

```diff
@@ -383,10 +382,12 @@ func (ap Approvers) GetCCs() []string {
 
 	currentApprovers := ap.GetCurrentApproversSet()
 	approversAndAssignees := currentApprovers.Union(ap.assignees)
-	suggested := ap.owners.KeepCoveringApprovers(approversAndAssignees, randomizedApprovers)
+	leafReverseMap := ap.owners.GetReverseMap(ap.owners.GetLeafApprovers())
+	suggested := ap.owners.KeepCoveringApprovers(leafReverseMap, approversAndAssignees, randomizedApprovers)
 	approversAndSuggested := currentApprovers.Union(suggested)
 	everyone := approversAndSuggested.Union(ap.assignees)
-	keepAssignees := ap.owners.KeepCoveringApprovers(approversAndSuggested, everyone.List())
```

Comment:
Oh, you need approversAndSuggested below.  This is very confusing and should be documented outside of the code.